adam-core/src/main/scala/org/bdgenomics/adam/rdd/fragment/InterleavedFASTQInFormatter.scala

```diff
@@ -59,11 +60,11 @@ class InterleavedFASTQInFormatter private (
     iter.flatMap(frag => {
       val reads = converter.convertFragment(frag).toSeq
 
-      if (reads.size < 2) {
+      if (enableLogging && reads.size < 2) {
```

Comment:
would simply dropping to `log.info` accomplish the same goal?

Reply:
I mean, it really is a warning message. However, if you're doing development of a piped tool and you want to read your executor logs to figure out why the tool running under the pipe is blowing up, the warnings get in the way of reading the logs.

If you're not 100% sold on this change, I'm OK with dropping it. It was just for my convenience when developing.

Reply:
I don't have data to test these particular checks (though I suppose I could invent some) and am not all that confident our logging configuration works.  Could you try adding
```
log4j.logger.org.bdgenomics.adam.rdd.fragment.InterleavedFASTQInFormatter=ERROR
```
to your log4j.properties and see if it does the trick?

Reply:
Yeah, agreed. I've rebased this and dropped the commit that resolves #1701.